Scott_Lee/week01homework/tuesday/scripts.js

```diff
@@ -0,0 +1,206 @@
+// What number's bigger?
+
+const greaterNum = function(a,b){
+  if (typeof(a) === `number` && typeof(b) === `number`){
+    if (a === b){
+      console.log(`Equal!`);
+    }
+    else console.log(`The greater number of ${a} and ${b} is ${(a > b ? a : b)}.`);
```

Comment:
I think you said earlier today that you feel strongly about writing your code in this format, but for the duration of the course can you please get into the habit of formatting this kind of if/ else statement in the following layout - 
if (a === b){
 console.log('Equal!') // if you are only outputting a string you do not need to use backticks ``
 } else {
 console.log(`The greater number of ${a} and ${b} is ${(a > b ? a : b)}.`);
 }